common/config/azure-pipelines/templates/gather-docs.yaml

```diff
@@ -99,6 +99,6 @@ steps:
       project: 2c48216e-e72f-48b4-a4eb-40ff1c04e8e4
       pipeline: 7436 # iTwin.js/Docs/iTwin.js Docs - YAML
       buildVersionToDownload: latestFromBranch
-      branchName: refs/heads/master
+      branchName: refs/heads/release/4.1.x
```

Comment:
Based on [this comment](https://github.com/iTwin/itwinjs-core/pull/5823#discussion_r1287232953), looks good